airbyte-integrations/connectors/source-mysql/metadata.yaml

```diff
@@ -22,54 +22,11 @@ data:
       enabled: true
     oss:
       enabled: true
-  releaseStage: generally_available
-  releases:
-    breakingChanges:
-      3.0.0:
-        message: Add default cursor for cdc
-        upgradeDeadline: "2023-08-17"
+  releaseStage: alpha
```

Comment:
@clnoll Should this return to alpha? I wasn't sure as source-mysql was already generally_available

Reply:
I'm not sure what the `releaseStage` refers to - it's not related to progressive rollouts. Maybe @alafanechere knows?

Reply:
Oh this is my favorite genre! /cc @nataliekwong and @alex-gron. TLDR I think is just keep it what it was :D

Reply:
I think it only affects internal reporting, not actual product behavior

Reply:
I'm not sure actually what this affects, sorry I can't be of more help!